Sources/Build/BuildOperation.swift

```diff
@@ -12,8 +12,6 @@
 
 import Basics
 
-import Build
```

Comment:
Unrelated warning, but probably not worth a separate one-liner PR, so chucking that in here instead.